tests/tools/hoppy/test_parser.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: UTF-8 -*-
+import mock
+import unittest
+
+from hamcrest import assert_that, has_entry, equal_to
+
+from ptp.tools.hoppy.parser import HoppyParser
+
+class TestHoppyParser(unittest.TestCase):
```

Comment:
Could you improve the tests for Hoppy? Add tests to check that it doesn't have false-positive when checking reports from other tools, etc.

See https://github.com/owtf/ptp/blob/develop/tests/tools/wapiti/test_parser.py for example.